### PR TITLE
Yet ANOTHER areas splicing PR

### PR DESCRIPTION
**Pull request by Jolly-66**
## About The Pull Request
Title.

## Why It's Good For The Game
Hi! Some time last year I was responsible for breaking up the original `space_station_13_areas.dm` ~~and responsible for breaking a few things in the process~~. Anyway- I was looking at one of Ghommies PRs and was snooping around the area files again.

This PR does two things:
1. Nukes `space_station_13_areas.dm` since it was an empty file, just containing a comment that was outdated.
2. Rips apart `areas/station.dm` into `areas/station/*`.

First and foremost, `station.dm` was a whopping `1,771` lines long, and that's just for areas used among our many maps of now and then.

I also deleted `/area/misc/asteroid` because it seemed unused after doing a search.

The old `station.dm` was converted into `areas/station/misc.dm` since not every single area is going to be created with organization in mind.

I also updated comments here and there to be a little more.. obvious? Also and lists I saw were expanded out. They look so much better now!!

## Changelog

N/A this shouldn't really affect anything. Muh organization.

### CODE REVIEW

**Comment by distributivgesetz** [comment]
my areas are undergoing mitosis again nooo

File: code/game/area/areas/station/cargo.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+* Cargo Areas
+*/
```
Comment (ZephyrTFA) [inline code/game/area/areas/station/cargo.dm:3]: get rid of these file headers

**Comment by timothymtorres** [comment]
This is great and should help prevent future merge conflicts. +1

**Comment by Jolly-66** [comment]
I'll get back to this in a few days, promise

**Comment by Jolly-66** [comment]
~~press the wrong button I hate Github~~

Addressed Zephyrs review.
However, for Solars and TComms I didn't remove the headers since they're more specific than just the department headers.
Security's header was condensed into a comment (since I never touched bounty code, I left it there).
I looked for a few more lists and expanded them out and added a trailing comma.

**Pull request merged**